README.md

```diff
@@ -1,2 +1,130 @@
+      DEPLOY_BRANCH: develop                                       # The branche for deploy
+      PYTHON_VERSION: 3.11                                         # The version of python
+      RUNNER: ubuntu-latest                                        # The type of github runner
+
+    secrets:
+      ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+
+```
+
+### Ansistrano deploy playbook
+
+```bash
+---
+- name: deployment
+  hosts: all
+  vars:
+    ansistrano_keep_releases: 10
+    ansistrano_deploy_from: "{{ playbook_dir }}/.."
+    ansistrano_deploy_to: "{{ deploy_path }}"
+    ansistrano_git_repo: "{{ deploy_repo }}"
+    ansistrano_git_branch: "{{ deploy_branch }}"
+    ansistrano_deploy_via: "git"
+    ansistrano_git_depth: 1
+    ansistrano_shared_paths:
+      - wp-content/uploads
+      - .well-known
+      - cgi-bin
+    ansistrano_shared_files:
+      - wp-config.php
+      - .htaccess
+  roles:
+      - { role: ansistrano.deploy }
+```
+`ansistrano_keep_releases` number of stored releases default 3
+
+`ansistrano_deploy_from` where my local project is (relative or absolute path)
+
+`ansistrano_deploy_to` base path to deploy to
+
+`ansistrano_git_repo` location of the git repository
+
+`ansistrano_git_branch` what version of the repository to check out. This can be the full 40-character SHA-1 hash, the literal string HEAD, a branch name, or a tag name
+
+`ansistrano_deploy_via` deployment strategy - method used to deliver code. Options are copy, download, git, rsync, rsync_direct, svn, or s3.
+
+`ansistrano_git_depth` additional history truncated to the specified number or revisions
+
+`ansistrano_shared_paths` shared folders, usually folders with images or other bulk data
+
+`ansistrano_shared_files` files that should not get into the repository are not necessary for the work of Wordpress
+
+#### Workflow for rollback for one step back
+
+```bash
+name: Rollback to Previously release
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    uses: saritasa-nest/saritasa-github-actions/.github/workflows/ci-wordpress.yaml@v2.4-dev.1
+    with:
+      ENVIRONMENT: production                                      # The Environment for deploy
+      SERVER_HOST: 50.87.253.20                                    # The pruduction server
+      SSH_DEPLOY_USER: ${{ vars.SSH_DEPLOY_USER }}                 # The username for connect to production server via ssh
+      PLAYBOOK_PATH: ci/ansible/ansistrano-rollback.yml              # The path to the file relative to the repository from which the deployment is made
+      DEPLOY_PATH: ~/deploy/                                       # The path to deploy on the production server
+      PYTHON_VERSION: 3.11                                         # The version of python
+      RUNNER: ubuntu-latest                                        # The type of github runner
+
+    secrets:
+      ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+
+```
+
+### Ansistrano rollback playbook
+
+```bash
+---
```

Comment:
if you reference `PLAYBOOK_PATH: ci/ansible/ansistrano-deploy.yml` then name this file accordingly vs just YAML snapshot.

Reply:
I Added filenames for examples section